force-app/main/default/classes/ODRIntegrationTest.cls

```diff
@@ -335,4 +335,38 @@ public class ODRIntegrationTest {
 
         System.assertEquals(new Map<String, Integration_Log__c>(), m1);
     }
+    
+    @isTest
+    public Static void fetchPrescriptionHistoryWithSearchKey_Test(){
```

Comment:
Please add negative test case scenarios for this test method.

Reply:
Added the Negative test scenario

Reply:
I don't see them, please review and submit PR again.